<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormMovieList</class>
 <widget class="QWidget" name="FormMovieList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>150</y>
     <width>640</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(100, 100, 100);</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelWifi">
   <property name="geometry">
    <rect>
     <x>1730</x>
     <y>1010</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMovie">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>125</width>
     <height>125</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/bds_UIs/MovieIcon128.PNG</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonReturn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMovieName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>770</y>
     <width>1901</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Resolution: 1920X1080</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelAirplay">
   <property name="geometry">
    <rect>
     <x>1660</x>
     <y>1010</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBluetooth">
   <property name="geometry">
    <rect>
     <x>1800</x>
     <y>1010</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBottom">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>980</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>harman/kardon</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelThumbnail">
   <property name="geometry">
    <rect>
     <x>1231</x>
     <y>270</y>
     <width>481</width>
     <height>431</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/bds_UIs/movie480.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="labelMovieText">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>410</y>
     <width>119</width>
     <height>48</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Movie</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
